packages/ui/src/components/typeahead/Typeahead.tsx

```diff
@@ -57,6 +58,7 @@ export const Typeahead: FunctionComponent<TypeaheadProps> = ({
   const onTextInputClear = () => {
     setInputValue('');
     setIsOpen(true);
+    onCleanInput();
```

Comment:
This object could be undefined when using the `TypeaheadField` outside of the `EnumField` scope.

```suggestion
    onCleanInput?.();
```